src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,24 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
+    private static final Comparator<Employee> AGE_NAME_COMPARATOR = new Comparator<>() {
+        @Override
+        public int compare(Employee o1, Employee o2) {
+            if (o1.getAge() != o2.getAge()) {
+                return o1.getAge() - o2.getAge();
```

Comment:
The general concept for comparators is to return either `0`, `1` or `-1`.
Because the `age` field is an integer, you can delegate the comparison to `Integer.compare()` method.